build_doc.sh

```diff
@@ -1,93 +1,71 @@
-#!/bin/bash
+#!/usr/bin/env bash
 set -e
 
-if [ "$#" -ne 1 ]; then
-	echo "./build_doc.sh <ktor_version>"
-	exit 1
-fi
-
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 KTOR_VERSION=$1
-CNAME=`cat CNAME`
+DOKKA_DIR=$2
 
-echo "Building KTOR_VERSION=$KTOR_VERSION..."
+if [ -z "${KTOR_VERSION}" ] || [ -z "${DOKKA_DIR}" ]; then
+  echo "Usage: $(basename "$0") version dokka-directory"
+  exit 1
+fi
 
-API_DOC_FOLDER="$PWD/doc-output"
+DOMAIN="$(cat "${DIR}/CNAME")"
+WORK_DIRNAME="output"
+WORK_DIR="${DIR}/${WORK_DIRNAME}"
 
-#######################################################################
+RESULT_DIRNAME="${KTOR_VERSION}"
+RESULT_DIR="${DIR}/${RESULT_DIRNAME}"
 
-JEKYLL_CONFIG="$(cat <<-EOF
+# Prepare working directory
+cp -rf "${DIR}/template" "${WORK_DIR}"
+cat > "${WORK_DIR}/_config.yml" <<- EOF
 
-# Jekyll configuration file
-title: "Ktor $KTOR_VERSION"
+title: "Ktor ${KTOR_VERSION}"
 description: Asynchronous framework for web applications
```

Comment:
If this is visible, let's make it the same as the text on ktor.io

Reply:
It's in the `<meta name="description"` tag

Reply:
Let's use the following: "Ktor is an asynchronous framework for creating microservices, web applications, and more."